scripts/determine-runc-version

```diff
@@ -36,33 +32,16 @@ runc_version() {
 	containerd_src_dir="${repo_abspath}/src/github.com/containerd/containerd"
 
 	if [ -f "${containerd_src_dir}/script/setup/runc-version" ]; then
-		# containerd v1.5.0-beta.4 and up, and v1.4.5 and up specify the version of
-		# runc to use in script/setup/runc-version.
+		# starting with v1.5.0-beta.4 and up, and v1.4.5, containerd specifies
+		# the version of runc to use in script/setup/runc-version.
 		runc_ref=$(cat "${containerd_src_dir}/script/setup/runc-version")
 		>&2 echo "INFO: detected runc version (${runc_ref}) from script/setup/runc-version"
 		echo "${runc_ref}"
 		return
-	elif [ -f "${containerd_src_dir}/go.mod" ]; then
-		# containerd master between v1.4.x and v1.5.0-beta.4 required the runc binary
-		# to be the same version as the vendored (libcontainer) dependency, specified
-		# in go.mod. containerd v1.5.0-beta.4 (and up), and v1.4.5 (and up) decoupled
-		# the binary version from the libnetwork version, and use script/setup/runc-version
-		runc_ref=$(grep 'opencontainers/runc' "${containerd_src_dir}/go.mod" | awk '{print $2}')
-		>&2 echo "INFO: detected runc version (${runc_ref}) from go.mod"
-		echo "${runc_ref}"
-		return
-	elif [ -f "${containerd_src_dir}/vendor.conf" ]; then
-		# containerd master between v1.4.x and v1.5.0-beta.4 required the runc binary
-		# to be the same version as the vendored (libcontainer) dependency, specified
-		# in vendor.conf.
-		runc_ref=$(grep 'opencontainers/runc' "${containerd_src_dir}/vendor.conf" | awk '{print $2}')
-		>&2 echo "INFO: detected runc version (${runc_ref}) from vendor.conf"
-		echo "${runc_ref}"
-		return
 	fi
 
 	# if all else fails
-	>&2 echo "INFO: unable to detect runc version, using HEAD"
+	>&2 echo "WARNING: unable to detect runc version, using HEAD"
 	echo "HEAD"
```

Comment:
As a follow-up, I want to change this into an error (also update `scripts/checkout.sh` to make both `$SRC` and `$REF` required (non-empty) arguments); https://github.com/docker/containerd-packaging/blob/c58c9172edf1b74c548e649a1282b008da7d43ef/scripts/checkout.sh#L17-L20